### PR TITLE
Add all files

### PR DESCRIPTION
**Pull request by susiku**
linter-error-flag

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+this will be ignored by git
```
Comment (Lordkaito) [inline test.md]: - [ ] Please make sure that this file is ignored by git, as this is not supposed to be in your repository.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @susiku, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
```
Comment (gemmen29) [inline README.md]: [OPTIONAL] Kindly remove this section as there is no live demo, you add this only if there exists a live demo for the project.

File: README.md [hunk 3]
```diff
@@ -1 +1,68 @@
...
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```
Comment (gemmen29) [inline README.md]: [OPTIONAL] Kindly remove this section, you add this only if there exists a `Prerequisites`, `Setup`, `Install`, `Usage`, `Run tests`, `Deployment` for the project.

File: README.md [hunk 4]
```diff
@@ -1 +1,68 @@
...
+## Authors
+
+👤 **wycliff susiku mubita*
+
+- GitHub: [@githubhandle](https://github.com/susiku)
+- Twitter: [@twitterhandle](https://twitter.com/@ws_mubita)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/wycliff-susiku-mubita-010423ba)
+
+👤
```
Comment (gemmen29) [inline README.md]: [OPTIONAL] Kindly remove this icon. as there is only one author in that project.

**Review by gemmen29 — APPROVED** [review]
Hi @susiku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions:
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**